scipy/stats/_sensitivity_analysis.py

```diff
@@ -405,10 +414,6 @@ def sobol_indices(
         consider at minima ``n >= 2**12``. The more complex the model is,
         the more samples will be needed.
 
-        If the parameters are not independent, the first-order indices would
-        not sum to 1.
-        Numerical noise can also contribute to this.
```

Comment:
Leave this last statement about noise

Reply:
What would you like it to say? I removed it since neither the first nor total order indices sum to 1 in general.

Reply:
Something along the line of: "Even for an additive model, indices could not sum to 1 due to numerical noise"